src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -172,10 +176,26 @@ private static bool IsDebuggerSpecialPunctuation(SyntaxKind kind)
 
         public static IEnumerable<SyntaxKind> GetPunctuationKinds()
         {
-            for (int i = (int)SyntaxKind.TildeToken; i <= (int)SyntaxKind.GreaterThanGreaterThanGreaterThanEqualsToken; i++)
+            for (int i = (int)SyntaxKind.TildeToken; i <= (int)SyntaxKind.DotDotToken; i++)
```

Comment:
>DotDotToken [](http://example.com/codeflow?start=74&length=11)

I think that in the SyntaxKind enum, around declarations of kinds that are hardcoded in implementation of this method, we should add comments describing what changes to the enum should lead to changes in this function as well.